app/src/main/java/com/codepath/apps/critterfinder/adapters/PetFavoritesAdapter.java

```diff
@@ -20,7 +20,7 @@
  */
 public class PetFavoritesAdapter extends RecyclerView.Adapter<PetFavoritesAdapter.ViewHolder> {
 
-    AdapterView.OnItemClickListener mOnItemClickListener;
+    private static OnItemClickListener mOnItemClickListener;
```

Comment:
I don't think we want the on item click listener to be static. We might use this pet list adapter in a couple different places.

I'd make it

private OnItemClickListener mOnItemClickListener;